src/main/scala/it/pagopa/interop/agreementprocess/error/AgreementProcessErrors.scala

```diff
@@ -73,4 +73,12 @@ object AgreementProcessErrors {
   final case class DescriptorNotFound(eServiceId: UUID, descriptorId: UUID)
       extends ComponentError("0014", s"Descriptor $descriptorId not found in EService $eServiceId")
 
+  final case class StampNotFound(stamp: String) extends ComponentError("0015", s"Agreement stamp $stamp not found")
+
+  final case class MissingUserInfo(user: UUID)
```

Comment:
```suggestion
  final case class MissingUserInfo(userId: UUID)
```
not to be pedantic 🤣 
Just a minor mental barrier to avoid that for some reason tomorrow types change and sensitive use info are passed here